client/my-sites/pages/page-card-info/index.jsx

```diff
@@ -70,6 +75,7 @@ function PageCardInfo( {
 
 	return (
 		<div className="page-card-info">
+			{ themeId && <QueryTheme siteId="wpcom" themeId={ themeId } /> }
```

Comment:
Non-blocking but we can instead relax `QueryTheme` to allow for a falsey value (which would no-op). This makes the client call cleaner, with just `<QueryTheme siteId="wpcom" themeId={ themeId } />`

Reply:
Good point, it might already do that, let me check

Reply:
`QueryTheme` marks it as a required prop

Reply:
Right but it doesn't need to be so strict, it can render null and bail early in that case.

Reply:
The component hasn't been changed in 2+ years and pretty much every instance where it gets used looks like this. I was hoping this was a change that wouldn't need too much API level changes.